hack/rpm-deps.sh

```diff
@@ -96,6 +96,7 @@ launcherbase_main="
 launcherbase_x86_64="
   edk2-ovmf-${EDK2_VERSION}
   qemu-kvm-device-usb-redirect-${QEMU_VERSION}
+  qemu-kvm-device-usb-host-${QEMU_VERSION}
```

Comment:
Do we have it also for arm?

Reply:
I see it here ~ https://mirror.stream.centos.org/9-stream/AppStream/aarch64/os/Packages/
Ah, I see, I should have added the arm rpm too. Thanks, I'll fix it in the next iteration (and then mark this as resolved)